apps/stats-dapp/src/containers/AuthoritiesTable/AuthoritiesTable.tsx

```diff
@@ -65,16 +65,19 @@ const columns: ColumnDef<AuthorityListItem, any>[] = [
   columnHelper.accessor('location', {
     header: 'Location',
     enableColumnFilter: false,
-    cell: (props) => (
-      <Typography
-        variant="h5"
-        fw="bold"
-        component="span"
-        className="!text-inherit"
-      >
-        {getUnicodeFlagIcon(props.getValue())}
-      </Typography>
-    ),
+    cell: (props) => {
+      const uniCode = props.getValue();
+      return (
+        <Typography
+          variant="body1"
+          fw="bold"
+          component="span"
+          className="!text-inherit"
+        >
+          {uniCode ? getUnicodeFlagIcon(uniCode) : 'Anonymous'}
```

Comment:
The fallback should be `Unknown` instead of `Anonymous` as the issue https://github.com/webb-tools/webb-dapp/issues/685 has described.